mining-paper.tex

```diff
@@ -296,14 +292,14 @@ \subsection{How much will I be compensated if I run a node?}
 But if you restake all your tokens, in the same first year you'd earn $177.5\%$ of the tokens you staked.
 
 \subsection{How many tokens will ever be in existence?}
-We'll start with $1$~billion tokens, and the maximum amount of tokens ever mined will be $3.89$~billion.
+We'll start with $1$~billion tokens, and the maximum amount of tokens ever minted and distributed will be $3.89$~billion.
 
 \subsection{What's the inflation rate?}
-The inflation rate will depend on how many short-term miners and long-term miners are working in the system.
-Depending on this, the initial inflation will be between $50\%$~APR (if all miners are very short term) and $100\%$~APR (if all miners commit for a long
+The inflation rate will depend on how many short-term stakers and long-term stakers are working in the system.
+Depending on this, the initial inflation will be between $50\%$~APR (if all stakers are very short term) and $100\%$~APR (if all stakers commit for a long
```

Comment:
For this statement and the next, is it a question of **all stakers are...** or **a sufficient number/majority of stakers are...**?